packages/flutter/lib/src/material/data_table.dart

```diff
@@ -47,9 +47,14 @@ class DataColumn {
   /// [Icon] (typically using size 18), or a [Row] with an icon and
   /// some text.
   ///
-  /// By default, this widget will only occupy the minimal space. If you want
-  /// it to take the entire remaining space, e.g. when you want to use [Center],
-  /// you can wrap it with an [Expanded].
+  /// This widget and the sort indicator (if visible) will wrap with a [Row],
+  /// by default, this widget will only occupy minimal space. It is
```

Comment:
The first sentence doesn't make much sense. Depending on what you mean, maybe

> This widget and the sort indicator are placed within a [Row], and by default, this widget will only occupy minimal space.

Reply:
As you have said [here](https://github.com/flutter/flutter/pull/51667#discussion_r391898431), this may break something.
In addition, folks may already wrap it under `Expanded`, and redundant `Expanded` will throw.

Reply:
It's astonishing how little I remember having said that at all.